bundles/org.openhab.binding.bloomsky/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,547 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bloomsky"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- BloomSky SKY 1/2 Personal Weather Station Thing -->
+	<thing-type id="sky">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge-api"/>
+		</supported-bridge-type-refs>
+		<label>Sky Weather Station</label>
+		<description>Provides weather data from the BloomSky personal weather stations account API</description>
+		<channel-groups>
+			<!-- Group for SKY 1/2 device details -->
+			<channel-group id="sky-device-information" typeId="sky-device-information">
+				<label>Sky Device Details</label>
+				<description>This is the Sky Weather Station device detail</description>
+			</channel-group>
+			<!-- Groups for SKY 1/2 weather observations -->
+			<channel-group id="sky-observations" typeId="sky-observations">
+				<label>Sky Observations</label>
+				<description>This is the Sky 1/2 Weather Station current weather observations</description>
+			</channel-group>
+			<channel-group id="sky-video-list" typeId="sky-video-list">
+				<label>Sky Video List°F </label>
+				<description>This is the SKY 1/2 Weather Station past five days of time-lapse videos in fahrenheit</description>
+			</channel-group>
+			<channel-group id="sky-video-list-c" typeId="sky-video-list-c">
+				<label>Sky Video List°C</label>
+				<description>This is the SKY 1/2 Weather Station past five days of time-lapse videos in celsius</description>
+			</channel-group>
+			<channel-group id="sky-preview-image-list" typeId="sky-preview-image-list">
+				<label>Sky Preview Images</label>
+				<description>This is the SKY 1/2 Weather Station last five preview images</description>
+			</channel-group>
+		</channel-groups>
+		<representation-property>deviceID</representation-property>
+
+	</thing-type>
+
+	<channel-group-type id="sky-device-information">
+		<label>Sky Device Detail</label>
+		<description>This is the Sky 1/2 device detail</description>
+		<channels>
+			<channel typeId="utc" id="utc"></channel>
```

Comment:
Could be simplified : `<channel typeId="utc" id="utc"/>`

Reply:
Cut and paste got me there; and thanks for the tip - made the necessary changes across all related xml definitions.